e2e/example.spec.ts

```diff
@@ -0,0 +1,18 @@
+import { test, expect } from '@playwright/test';
+
+test('has title', async ({ page }) => {
+  await page.goto('https://playwright.dev/');
+
+  // Expect a title "to contain" a substring.
+  await expect(page).toHaveTitle(/Playwright/);
```

Comment:
The first basic test would be

1. open firecamp.dev
2. open rest request playground
3. test json `https://jsonplaceholder.typicode.com/todos/1`
4. asset the response